### DISCUSSION TITLE
FDS performance with flame engulfment

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12820] [General]
I was reading through the FDS validation scenario and couldn't find much useful. I'm wondering if anyone knows how FDS performs when attempting to calculate heat flux to targets when the flame is impinging or engulfing the target. I'm specifically looking for how it performs for non-jet fire scenarios. Any advice or insight is greatly appreciated.

**Comment by mcgratta** [comment]
FM_Vertical_Wall_Flames
Hasemi_LocFire_Column
NIST_Composite_Beam 
NIST_E119_Compartment
NIST_FSE_2008

**Comment by mcgratta** [comment]
NIST_NRC_Parallel_Panels
NRCC_Facade
SP_AST
WTC
